packages/coding-standards/tests/Unit/CsFixer/OrmJoinColumnRequireNullableFixer/fixed/fixed.php

```diff
@@ -0,0 +1,11 @@
+<?php
```

Comment:
When we have here more wrong/fixed files, please rename them so they express what is wrong in the filename
This can be for example `fixed_many_to_one_missing_join_column`

And the same can be done with correct cases... Because correct3 doesn't tell anything